src/test/scala-2.12/org/apache/james/gatling/simulation/SimulationOnMailCorpus.scala

```diff
@@ -26,6 +27,7 @@ trait SimulationOnMailCorpus {
     .map(new MailboxName(_))
 
   protected val feeder: SourceFeederBuilder[String] = UserFeeder.toFeeder(getUsers).circular
+  protected val recipientFeeder: RecipientFeederBuilder = RecipientFeeder.usersToFeeder(getUsers)
```

Comment:
Why cant we reuse feeder like on draft?

Reply:
What do you mean? It is the same as draft?

Reply:
Not exactly... In the draft simulation, we don't depend on the provisioning (does not extend `SimulationOnMailCorpus`). We don't need the provisioning here I believe, you shouldn't need to touch this file I think